web_selection_autocomplete/static/src/scss/web_selection_autocomplete.scss

```diff
@@ -0,0 +1,9 @@
+.o_selection_autocomplete_invalid {
+    &::before {
+        color: #DF382C !important;
+        content: 'Wrong value. Insert one of the available options.' !important;
```

Comment:
Is there a need for this? Can we use the already existing mechanism (ServiceProviderMixin.do_warn) to inform the user that their input is invalid?